tests/Behat/Gherkin/Cache/FileCacheTest.php

```diff
@@ -7,7 +7,7 @@
 use Behat\Gherkin\Node\ScenarioNode;
 use Behat\Gherkin\Gherkin;
 
-class FileCacheTest extends \PHPUnit_Framework_TestCase
+class FileCacheTest extends \PHPUnit\Framework\TestCase
```

Comment:
I suggest adding a use statement instead for consistency with the coding standards.

Reply:
Done in latest commit.